gpii/node_modules/canopyMatchMaker/src/CanopyMatchMakerUtilities.js

```diff
@@ -142,37 +145,19 @@ gpii.canopyMatchMaker.utils.prefixLength = function (path, model) {
     return 0;
 };
 
-// only used by canopyMatchMaker - moved from MatchMakerFramework
-gpii.canopyMatchMaker.utils.sortDescending = function (numA, numB) {
-    return numB - numA;
-};
+fluid.registerNamespace("gpii.canopyMatchMaker.utils.ISO24751");
 
-// TODO: UNUSED FUNCTION
-gpii.canopyMatchMaker.utils.makeComparator = function (leaves) {
-    return function (solnA, solnB) {
-        var domA = false, domB = false;
-        for (var i = 0; i < leaves.length; ++i) {
-            var leaf = leaves[i];
-            var goodA = gpii.canopyMatchMaker.utils.prefixLength(leaf, solnA.solution);
-            var goodB = gpii.canopyMatchMaker.utils.prefixLength(leaf, solnB.solution);
-            if (goodA > goodB) {
-                domA = true;
-            }
-            else if (goodB > goodA) {
-                domB = true;
-            }
-        }
-        return domA ^ domB ? (domA ? 1 : -1) : 0;
-    };
+gpii.canopyMatchMaker.utils.ISO24751.prefixLength = function (path, model) {
```

Comment:
This can be named gpii.canopyMatchMaker.utils.ISO24751.ontologicalMetric since it no longer strictly computes prefixLength